schemas/Module.yaml

```diff
@@ -0,0 +1,42 @@
+title: Module
+type: object
+required:
+  - title
+  - lessons
+  - instructors
```

Comment:
`instructors` property is a property of a course and as such won't be required here.